db/CSV_Writers/config/provisions.js

```diff
@@ -0,0 +1,71 @@
+const {
+  packages: { path },
+  random: { generateInBetweenSync: gIBS, randomString: rS, twoValProb: tVP, threeValProb: rEVP },
+  userConstraints, reviewConstraints
+} = require('./config.js');
+
+module.exports = {
+  userProvisions: {
+    title: 'users',
+    constraints: userConstraints,
+    template: () => `${"'" + rS() + "'"},${module.exports.userProvisions.jsonFile[(gIBS(250))]},${gIBS(10)},${gIBS(100)},${gIBS(20)},${gIBS(125)},${gIBS(80)},${gIBS(2)}` + '\r\n',
+    jsonFile: require(path.resolve(`Read_Write_Files/profileLinks.json`)),
+    dBQueryStr: `COPY users(userName, profilePicture, userTheme, steamLevel, reviewsGiven, playtime, productActivation, gamesOwned) FROM '${path.resolve('Read_Write_Files/records.csv')}' DELIMITER ','`,
+    dBResetStr: `
+      DROP TABLE IF EXISTS users;
+      CREATE TABLE users (
+      id SERIAL PRIMARY KEY,
+      userName varchar,
+      profilePicture varchar,
+      userTheme int,
+      steamLevel int,
+      reviewsGiven int,
+      playtime int,
+      productActivation int,
+      gamesOwned int
+    );`,
+    dBCountStr: `SELECT COUNT(*) FROM users`,
```

Comment:
this is cool. i would never have thought to store the SQL statements as strings in objects. very clever way to compartmentalize code.

Reply:
Thank you!I I gotta remove the dBCountStr since that strategy didn't end up working with postgres, and its a little over kill for a one liner, so i'll do it gladly